drivers/wifi/eswifi/eswifi_bus_spi.c

```diff
@@ -202,7 +202,7 @@ int eswifi_spi_init(struct eswifi_dev *eswifi)
 	struct eswifi_spi_data *spi = &eswifi_spi0; /* Static instance */
 
 	/* SPI DEV */
-	spi->spi_dev = device_get_binding("SPI_3");
+	spi->spi_dev = device_get_binding(ESWIFI0_BUS_NAME);
```

Comment:
where is ESWIFI0_BUS_NAME coming from?

Reply:
Generated from dts.

Reply:
shouldn't this be DT_something?

Reply:
In theory, but we also generate a set of defines based on "aliases" so we have:

```
#define ESWIFI0_BASE_ADDRESS                                                            DT_ST_STM32_SPI_FIFO_40003C00_INVENTEK_ESWIFI_1_BASE_ADDRESS
#define ESWIFI0_BUS_NAME                                                                DT_ST_STM32_SPI_FIFO_40003C00_INVENTEK_ESWIFI_1_BUS_NAME
...
```

I'll push a fix to clean that up as part of this PR.